sdk/storage/storage-blob-changefeed/.vscode/settings.json

```diff
@@ -1,27 +0,0 @@
-{
-    "files.exclude": {
-      "**/.git": true,
-      "**/.svn": true,
-      "**/.DS_Store": true
-    },
-    "[typescript]": {
-      "editor.formatOnSave": true,
-      "editor.tabSize": 2,
-      "editor.detectIndentation": false
-    },
-    "[json]": {
-      "editor.formatOnSave": false,
-      "editor.tabSize": 2,
-      "editor.detectIndentation": false
-    },
-    "[yaml]": {
-      "editor.formatOnSave": true,
-      "editor.tabSize": 2,
-      "editor.detectIndentation": false
-    },
-    "editor.rulers": [
-      100
-    ],
-    "typescript.preferences.quoteStyle": "double",
-    "javascript.preferences.quoteStyle": "double"
```

Comment:
>"javascript.preferences.quoteStyle": "double" [](start = 4, length = 45)

I don't think you should remove this in this PR before git format hook is added, it's not related to debug and difficult for future tracking.

The settings are helpful for anyone contributing to storage code base with double quotes and 2 tabSize. You can remove these in another PR which adds git format hook.

Reply:
Done, please take a look: https://github.com/Azure/azure-sdk-for-js/pull/10219

Reply:
I believe this has been resolved now that #10219 is about to be merged.